lantern_external_index/Cargo.toml

```diff
@@ -13,7 +13,8 @@ clap = { version = "4.4.0", features = ["derive"] }
 cxx = "1.0.106"
 postgres = "0.19.7"
 postgres-types = { version = "0.2.6", features = ["derive"] }
-usearch = { git = "https://github.com/Ngalstyan4/usearch.git", branch = "main-lantern" }
+# usearch = { git = "https://github.com/Ngalstyan4/usearch.git", branch = "main-lantern" }
+usearch = {path = "../../third_party/usearch/"}
```

Comment:
the version I used is now upstream @ narek/pq-index https://github.com/Ngalstyan4/usearch/tree/narek/pq-index